packages/context/test/unit/context.ts

```diff
@@ -213,16 +205,8 @@ describe('Context', () => {
       const SEP = Binding.PROPERTY_SEPARATOR;
       const val = {x: {y: 'Y'}};
       ctx.bind('foo').to(Promise.resolve(val));
-      let result = await ctx.get('foo');
-      expect(result).to.equal(val);
-      result = await ctx.get(`foo${SEP}x`);
-      expect(result).to.eql({y: `Y`});
-      result = await ctx.get(`foo${SEP}x.y`);
```

Comment:
What is the expected behavior of `ctx.get(`foo${SEP}x.y`);` now?

Reply:
Same as before, see unit-tests for `Binding`: https://github.com/strongloop/loopback-next/blob/94c3b19bd501ea909e8676a57254a618d1c69451/packages/context/test/unit/binding.ts#L101-L129

Since `ctx.get` is using the implementation in Binding, I feel we should not be replicating `Binding` tests, in order to keep our maintenance costs low.